shipyard-controller/internal/provisioner/repository_provisioner.go

```diff
@@ -53,8 +53,8 @@ func (rp *RepositoryProvisioner) ProvideRepository(projectName, namespace string
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode == http.StatusConflict {
-		return nil, fmt.Errorf(common.UnableProvisionInstance, http.StatusText(http.StatusConflict))
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Comment:
Maybe is worth to log the response data. I am thinking about a 302, this way an operator can follow through the redirect. 
WDYT?